tests/testChebyValues.py

```diff
@@ -148,10 +159,10 @@ def setUp(self):
     def tearDown(self):
         del self.orbits
         del self.jpl
-        os.remove(self.coeffFile)
-        os.remove(self.residFile)
-        if os.path.isfile(self.failedFile):
-            os.remove(self.failedFile)
+        #os.remove(self.coeffFile)
```

Comment:
Did you mean to leave these commented-out?  They look like clean-up that should be done.